lib/Elevate/Blockers/EA4.pm

```diff
@@ -72,14 +75,33 @@ sub _get_incompatible_packages ($self) {
         if ( $pkg =~ m/^(ea-php[0-9]+)/ ) {
             my $php_pkg = $1;
             next unless $self->_php_version_is_in_use($php_pkg);
+
+            if ( $self->_php_is_provided_by_imunify_360($php_pkg) ) {
+                push @imunify_pkgs, $pkg;
+                next;
+            }
         }
 
         push @incompatible, $pkg;
     }
 
+    if (@imunify_pkgs) {
+        Elevate::StageFile::update_stage_file( { ea4_imunify_packages => \@imunify_pkgs } );
+    }
+
     return @incompatible;
 }
 
+sub _php_is_provided_by_imunify_360 ( $self, $php ) {
+    return 0 unless -x Elevate::Constants::IMUNIFY_AGENT;
```

Comment:
this is only there if 360 is present?

Reply:
I'm not sure about this, but I think it might be any Imunify product.  This was just what `Elevate::Components::Imunify::is_installed` was using to determine if it was installed, so I used it here.  It should not hurt anything if we let things through with regular Imunify installed, but I could look for a better way to make sure it is Imunify 360 specifically if preferred.  Thoughts?